readme.md

```diff
@@ -1,27 +1,23 @@
-## WeePickle
+# WeePickle [![Build Status](https://travis-ci.org/rallyhealth/weePickle.svg)](https://travis-ci.org/rallyhealth/weePickle) <img src="https://api.bintray.com/packages/rallyhealth/maven/weePickle/images/download.svg" title="Download" />
+
 A JSON, YAML, MsgPack, XML, etc. serialization framework with MiMa and shading.
-Safe for use by libraries without causing dependency hell.
+
```

Comment:
differences would be worthwhile to call out _somewhere_ highly visible... here's a potential spot for it. Perhaps we could put it in the repo's description? I thought that we had a "motivation" section at one point, but I can't seem to find it.

<img width="500" alt="Screen Shot 2020-02-05 at 8 51 56 AM" src="https://user-images.githubusercontent.com/20071387/73847725-d3d89400-47f4-11ea-8d93-8c85bbc134ff.png">

```suggestion

Fork of [uPickle](https://github.com/lihaoyi/uPickle). See [differences](differences.md).

```